### ISSUE TITLE
Add and display Internet Radio information

### ISSUE THREAD

**Issue by fenix-hub** [fenix-hub/RadioNet#3]
Find a way to display some additional information or metadata of the radio station you are currently playing.

*Possible solutions:*  
- Even though the radio station streaming is done via a .dll, HTTPRequests can be done on the internet radio station link anyway.  
e.g. make a GET HTTPRequest on `http://shoutcast.rtl.it:3010/`, fetch all the data contained in the response, extract and display some useful information.  
-  NAudio.dll, which is the library used to stream internet radio stations playing, has some methods to fetch datas about a specific station. Find it [here](https://github.com/naudio/NAudio)

**Comment by JhoanAlvear** [comment]
can i help

**Comment by fenix-hub** [comment]
Yes, feel free to contribute for the hacktoberfest, the repository is open to PRs

**Comment by JhoanAlvear** [comment]
assign issue please
